src/Optimism.sol

```diff
@@ -3,6 +3,57 @@ pragma solidity >=0.8.0;
+    /*** Bridging Addresses                                                                                         ***/
+    /******************************************************************************************************************/
+
+    address internal constant CCTP_TOKEN_MESSENGER = 0x2B4069517957735bE00ceE0fadAE88a26365528f;
+
+    address internal constant SKY_GOV_RELAY = 0x10E6593CDda8c58a1d0f14C5164B376352a55f2F;
+    address internal constant TOKEN_BRIDGE  = 0x8F41DBF6b8498561Ce1d73AF16CD9C0d8eE20ba6;
+
+    /******************************************************************************************************************/
+    /*** PSM Addresses                                                                                              ***/
+    /******************************************************************************************************************/
+
+    address internal constant PSM3 = 0xe0F9978b907853F354d79188A3dEfbD41978af62;
+
+    /******************************************************************************************************************/
+    /*** Spark Liquidity Layer Addresses                                                                            ***/
+    /******************************************************************************************************************/
+
+    address internal constant ALM_CONTROLLER  = 0x1d54A093b8FDdFcc6fBB411d9Af31D96e034B3D5;
+    address internal constant ALM_PROXY       = 0x876664f0c9Ff24D1aa355Ce9f1680AE1A5bf36fB;
+    address internal constant ALM_RATE_LIMITS = 0x6B34A6B84444dC3Fc692821D5d077a1e4927342d;
+
+    address internal constant ALM_FREEZER  = 0x90D8c80C028B4C09C0d8dcAab9bbB057F0513431;
+    address internal constant ALM_RELAYER  = 0x8a25A24EDE9482C4Fc0738F99611BE58F1c839AB;
+    address internal constant ALM_RELAYER2 = 0x8Cc0Cb0cfB6B7e548cfd395B833c05C346534795;
+
+    /******************************************************************************************************************/
+    /*** Governance Relay Addresses                                                                                 ***/
+    /******************************************************************************************************************/
+
+    address internal constant SPARK_EXECUTOR = 0x205216D89a00FeB2a73273ceecD297BAf89d576d;
+    address internal constant SPARK_RECEIVER = 0x61Baf0Ce69D23C8318c786e161D1cAc285AA4EA3;
+
+    /******************************************************************************************************************/
+    /*** SSR Oracle Addresses                                                                                       ***/
+    /******************************************************************************************************************/
+
+    address internal constant SSR_AUTH_ORACLE             = 0x6E53585449142A5E6D5fC918AE6BEa341dC81C68;
```

Comment:
Deployer does not have admin
SSR_RECEIVER has DATA_PROVIDER_ROLE
```
forge verify-bytecode 0x6E53585449142A5E6D5fC918AE6BEa341dC81C68 SSRAuthOracle
Verifying bytecode for contract SSRAuthOracle at address 0x6E53585449142A5E6D5fC918AE6BEa341dC81C68
Creation code matched with status full
Runtime code matched with status full
```

Reply:
Compare against arbitrum SSR oracle getSUSDSData (same SSR, different chi and rho values):

```
arb: 1000000001395766281313196627,1051101385182530708425233324,1747057583
opt: 1000000001395766281313196627,1048457366574888685631260128,1745253095
```

getAPR (same resulting value)

```
arb: 44016885447492968829072000 
opt: 44016885447492968829072000
```